src/components/Authentication/ForgotPassword.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { sendPasswordReset } from '../../common/auth_utils';
+
+const ForgotPassword = () => {
+  const [email, setEmail] = useState();
+  const [errorMessage, setErrorMessage] = useState();
+  const [confirmationMessage, setConfirmationMessage] = useState();
+  const handleForgotPassword = async e => {
+    try {
+      e.preventDefault();
+      await sendPasswordReset(email);
+      setConfirmationMessage(
+        'If the email entered is associated with an account, you should receive an email to reset your password shortly.',
+      );
```

Comment:
Can we tell users if an email they entered to reset a password exists or not? Like if it doesn't exist we should say "uh oh you don't exist" or something